appveyor.yml

```diff
@@ -0,0 +1,33 @@
+os:
+  - "WMF 5"
+
+branches:
+  only:
+    - master
+
+install:
+  - ps: |
+      Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force | Out-Null
+      Install-Module Pester -MinimumVersion 3.4.0 -Scope CurrentUser -Force | Out-Null
+
+build_script:
+  - ps: |
+      # Nothing to build at this point, just run Pester tests
+      $testResultsFile = Join-Path $pwd TestResults.xml
+      $testing = @{
+          Script       =
```

Comment:
Build failed because no value is specified here:

> OutputFile : The term 'OutputFile' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.